src/DacpacTool/PackageBuilder.cs

```diff
@@ -161,7 +161,7 @@ public void SaveToDisk(FileInfo outputFile, PackageOptions packageOptions = null
             EnsureModelCreated();
             EnsureModelValidated();
             EnsureMetadataCreated();
-            
```

Comment:
Looks like there are a bunch of unnecessary whitespace changes. Not the end of the world, but it would be nice to fix these.

Reply:
Fixed. This was due to my Rider settings